cf/s3.yaml

```diff
@@ -48,14 +48,3 @@ Resources:
         BlockPublicPolicy: True
         IgnorePublicAcls: True
         RestrictPublicBuckets: True
-
-  # The bucket that contains the original unfiltered versions of the count matrix files used for the experiments.
-  UploadedSamples:
-    Type: AWS::S3::Bucket
-    Properties:
-      BucketName: !Sub "biomage-uploaded-samples-${Environment}"
-      PublicAccessBlockConfiguration:
-        BlockPublicAcls: True
-        BlockPublicPolicy: True
-        IgnorePublicAcls: True
-        RestrictPublicBuckets: True
```

Comment:
This bucket is no longer necessary, we can put up people's samples directly into `biomage-originals`. This was only created as a way to not have to worry about people with access to the platform uploading stuff to S3 without oversight.